easybuild/easyconfigs/m/Molden/Molden-7.1-GCCcore-11.3.0.eb

```
@@ -0,0 +1,48 @@
+easyblock = 'MakeCp'
+
+name = 'Molden'
+version = '7.1'
+
+homepage = 'https://www3.cmbi.umcn.nl/molden'
```

```suggestion
homepage = 'https://www.theochem.ru.nl/molden/'
```
I get a 404 for the original. I think the suggested one is the correct replacement.

---

done